play-json/jvm/src/main/scala/play/api/libs/json/BigNumberParser.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package play.api.libs.json
+
+class BigNumberParser(settings: JsonParserSettings) {
+
+  import BigNumberParser._
+
+  def parseBigDecimal(input: String): java.math.BigDecimal = {
```

Comment:
Would prefer

```suggestion
  def parseBigDecimal(input: String): Try[java.math.BigDecimal] = {
```

+ `JsResult.fromTry`

Or directly `JsResult`

Reply:
`parseBigDecimal` is used in the json parser as well. Exceptions are expected to be thrown there.